examples/pytorch/graphsage/dist/train_dist.py

```diff
@@ -1,29 +1,17 @@
-import os
-
-os.environ["DGLBACKEND"] = "pytorch"
 import argparse
-import math
 import socket
 import time
-from functools import wraps
-from multiprocessing import Process
+from contextlib import contextmanager
 
 import numpy as np
 import torch as th
-import torch.multiprocessing as mp
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.optim as optim
 import tqdm
-from torch.utils.data import DataLoader
-
 import dgl
-import dgl.function as fn
 import dgl.nn.pytorch as dglnn
-from dgl import DGLGraph
-from dgl.data import load_data, register_data_args
-from dgl.data.utils import load_graphs
-from dgl.distributed import DistDataLoader
+from dgl.data import register_data_args
```

Comment:
Is this being used? same for other scripts

Reply:
it's called but I think it's not necessary. we could remove it as we always use `graph_name` to identify the dataset.